mmdet3d/datasets/kitti_dataset.py

```diff
@@ -32,12 +32,18 @@ class KittiDataset(Det3DDataset):
             - 'LiDAR': Box in LiDAR coordinates.
             - 'Depth': Box in depth coordinates, usually for indoor dataset.
             - 'Camera': Box in camera coordinates.
-        filter_empty_gt (bool): Whether to filter the data with empty GT.
-            If it's set to be True, the example with empty annotations after
-            data pipeline will be dropped and a random example will be chosen
-            in `__getitem__`. Defaults to True.
         test_mode (bool): Whether the dataset is in test mode.
             Defaults to False.
+        filter_cfg (dict, optional): Config for filter data.
+            the filter keys of kitti dataset include:
+                - filter_dontcare: bool, whether to filter the 'DontCare'
+                    objects in kitti dataset.
+                - filter_class: bool, whether to filter not required classes
+                    (classes not in metainfo).
+                - filter_empty_gt: bool, whether to filter the data sample
+                    without annotations.
+            Defaults to dict(filter_class=False, filter_empty_gt=False
```

Comment:
Better to add some description about optional keys used in mono3d: filter_by_truncation, filter_by_occlusion, filert_min_size